src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+
+    'coords': {
```

Comment:
```suggestion
    version,
    coords: {
```
Keys of the object are always strings